powerauth-nextstep-client/src/main/java/io/getlime/security/powerauth/lib/nextstep/client/NextStepClient.java

```diff
@@ -96,8 +96,8 @@ private RestTemplate defaultTemplate() {
      * @param params        list of generic parameters
      * @return a Response with CreateOperationResponse object for OK status or ErrorModel for ERROR status
      */
-    public ObjectResponse<CreateOperationResponse> createOperation(String operationName, String operationId, String operationData, List<KeyValueParameter> params) throws NextStepServiceException {
-        return createOperation(operationName, operationId, operationData, null, params);
+    public ObjectResponse<CreateOperationResponse> createOperation(String operationName, String operationId, String operationData, String httpSessionId, List<KeyValueParameter> params) throws NextStepServiceException {
```

Comment:
This will not work. How does system outside the browser know about HTTP session? Method `createOperation` is called from outside the web browser most of the time (payment is created via microservice adapter running at bank back-end)...

Reply:
(In other words: HTTP session must be checked somewhere on webflow level.)